cmd/plugin/rpaasv2/cmd/certificate.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tsuru/rpaas-operator/cmd/plugin/rpaasv2/proxy"
+)
+
+func init() {
+	rootCmd.AddCommand(certificateCmd)
+
+	certificateCmd.Flags().StringP("service", "s", "", "Service name")
+	certificateCmd.Flags().StringP("instance", "i", "", "Service instance name")
+	certificateCmd.Flags().StringP("certificate", "c", "", "Certificate file name")
+	certificateCmd.Flags().StringP("key", "k", "", "Key file name")
```

Comment:
On RPaaS v2 web API, we're accepting a new field `name` which gives a name to the provided certificate-key, it defaults to `default` value when not sent. Can you add this?

Reply:
On it.